src/utils/BlockExplorerUtils.ts

```diff
@@ -2,53 +2,53 @@ import { PUBLIC_NETWORKS, DEFAULT_BLOCKCHAIN_EXPLORER_DOMAIN } from "../constant
 import { createShortHexString } from "./FormattingUtils";
 
 /**
- * Creates an etherscan link for a transaction or address on a given network.
+ * Creates a block explorer link for a transaction or address on a given network.
  * @param txHashOrAddress The transaction hash or address to link to.
  * @param chainId The chainId to link to.
- * @returns The etherscan link. This link will be formatted in markdown.
+ * @returns The block explorer link. This link will be formatted in markdown.
  */
-export function etherscanLink(txHashOrAddress: string, chainId: number | string): string {
-  return _createEtherscanLinkMarkdown(txHashOrAddress, Number(chainId)) ?? "<>";
+export function blockExplorerLink(txHashOrAddress: string, chainId: number | string): string {
+  return _createBlockExplorerLinkMarkdown(txHashOrAddress, Number(chainId)) ?? "<>";
 }
 
 /**
- * Resolves a domain to an etherscan link for a given network.
+ * Resolves a domain to an block explorer link for a given network.
  * @param networkId The network to link to.
- * @returns The etherscan link. If the networkId is not supported, the default etherscan mainnet link will be returned.
+ * @returns The block explorer link. If the networkId is not supported, the default block explorer mainnet link will be returned.
  */
-export function createEtherscanLinkFromTx(networkId: number): string {
+export function resolveBlockExplorerDomain(networkId: number): string {
   return PUBLIC_NETWORKS[networkId]?.etherscan ?? DEFAULT_BLOCKCHAIN_EXPLORER_DOMAIN;
 }
```

Comment:
Minor observation for future consideration (i.e. probably not to be changed in this PR): the existing behaviour of defaulting to etherscan doesn't make sense to me, since if we don't recognise the chainId then any etherscan link will just resolve to an etherscan error page.

I think it'd be fine to throw (or return undefined) and then handle that within the caller to produce some meaningful text (...and include the txn hash).

Reply:
Done: https://github.com/across-protocol/sdk-v2/pull/378/commits/31f986b365bc6f606e71a6d009418841a0d85df5